packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.ts

```diff
@@ -95,6 +105,7 @@ export class WizardStepHeaderComponent extends CdkStepHeader implements AfterVie
 
     ngAfterViewInit(): void {
         this._focusMonitor.monitor(this._elementRef, true);
+        this.detectLongLabel();
```

Comment:
Do we need handle dynamic changes in the available space and label length? If so, this would need to be done in ngOnChanges as well.

Reply:
I guess we don't need to handle width dynamically. It's really edge case if user starts resize browser window. And dom calculation for this could cost performance. Other case will be cover by NUI-5068.

Reply:
Actually, I don't think we should consider resizing to be an edge case. If we have logic to show a tooltip on overflow, I think it's important that the logic works without having to refresh the page.

Reply:
On the other hand, if UX agrees that we can unconditionally show the tooltip regardless of overflow status, this problem would go away.